packages/strapi-plugin-content-manager/admin/src/containers/ListView/ListViewLayout.js

```diff
@@ -2,14 +2,15 @@ import React, { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 import { useQueryParams } from 'strapi-helper-plugin';
-import RBACManager from '../RBACManager';
-import { setLayout } from './actions';
+import { resetProps, setLayout } from './actions';
 import ListView from './index';
+import useSyncRbac from '../RBACManager/useSyncRbac';
 
 const ListViewLayout = ({ layout, ...props }) => {
   const dispatch = useDispatch();
   const initialParams = useSelector(state => state.get('content-manager_listView').initialParams);
-  const [, setQuery] = useQueryParams(initialParams);
+  const [{ query }, setQuery] = useQueryParams(initialParams);
+  const permissions = useSyncRbac(query, props.slug);
```

Comment:
When dealing with the ListView in the CM, it deals with a lot a things. In fact, many things are actually coupled to the ListView lifecycle.

Now, with i18n, we need to destroy/re-create the listview based on the locale that exists in the query string.

In order to do so, the permission check has been moved higher in the React tree. Now, the ListViewLayout is the one responsible for dealing with the permissions and the ListView only consumes them.